src/manifest.c

```diff
@@ -1010,7 +1031,7 @@ struct list *recurse_manifest(struct manifest *manifest, struct list *subs, cons
 			version1 = version2;
 		}
 
-		sub = load_manifest(version1, version2, file, manifest, false);
+		sub = load_manifest(version1, version2, file, manifest, false, delta_manifest);
```

Comment:
This attempts to load bundles that have not changed. The load_manifest function will fail to find an unchanged delta manifest and fallback to downloading the entire bundle manifest. I will add a case to skip bundles that have not changed when looking for delta_manifests.